docs/design.html

```diff
@@ -531,9 +531,17 @@ <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring
     <pre class="brush: text;">  log.cleaner.min.compaction.lag.ms</pre>
 
     This can be used to prevent messages newer than a minimum message age from being subject to compaction. If not set, all log segments are eligible for compaction except for the last segment, i.e. the one currently
-    being written to. The active segment will not be compacted even if all of its messages are older than the minimum compaction time lag.
+    being written to.
     </p>
     <p>
+    The log cleaner does not clean the active log segment. Therefore pick a suitable time or size to roll the segment file when using compaction.</p>
+    <pre class="brush: text;">  segment.ms</pre>
```

Comment:
If we mention `segment.ms`, can we either mention explicitly that this is the topic-level config or can we also mention `log.roll.ms`?